src/Serialise/Cborg/Tutorial.hs

```diff
@@ -16,33 +16,239 @@ module Serialise.Cborg.Tutorial
     -- ** The CBOR format
     -- $cborformat
 
+    -- ** The 'Serialise' class
+    -- $serialise
+
     -- ** Encoding terms
     -- $encoding
 
     -- ** Decoding terms
     -- $decoding
+
+    -- ** Migrations
+    -- $migrations
   ) where
 
+import Serialise.Cborg
+
 {- $introduction
 
-Lorem ipsum...
+@cborg@ is a library for the serialisation of Haskell values.
+
+As in modern serialisation libraries, @cborg@ offers
+instance derivation via GHC's 'GHC.Generic' mechanism,
+
+> import Serialise.Cborg
+> import qualified Data.ByteString.Lazy as BSL
+>
+> data Animal = HoppingAnimal { animalName :: String, hoppingHeight :: Int }
+>             | WalkingAnimal { animalName :: String, walkingSpeed :: Int }
+>             deriving (Generic)
+>
+> instance Serialise Animal
+>
+> fredTheFrog :: Animal
+> fredTheFrog = HoppingAnimal "Fred" 4
+>
+> main = BSL.writeFile "hi" (serialise fredTheFrog)
+
+We can then later read Fred,
+
+> main = do
+>     fred <- deserialise <$> BSL.readFile "hi"
+>     print fred
 
 -}
 
 {- $cborformat
 
-Lorem ipsum...
+@cborg@ uses the Concise Binary Object Representation, CBOR
+(IETF RFC 7049), as its serialised representation. This encoding is efficient in
+both encoding\/decoding complexity as well as space, and is generally
+machine-independent.
+
+The CBOR data model resembles that of JSON, having arrays, key\/value maps,
+integers, floating point numbers, binary strings, and text. In addition,
+CBOR allows items to be /tagged/ with a number which describes the type of data
+that follows. This can be used both to identify which data constructor of a type
+an encoding represents, as well as representing different versions of the same
+constructor.
+
+*** A note on interoperability
```

Comment:
This line doesn't seem to format properly in Haddock, showing as a list element followed by `**  A note on interoperability`